src/stcal/ramp_fitting/ramp_fit.py

```diff
@@ -16,7 +16,7 @@
 import numpy as np
 import logging
 
-from . import constants
+# from . import constants
```

Comment:
If commented out, should these lines just be removed?

Reply:
Yes.  I removed the comment.